src/test/regress/bin/diff

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+file1="${@:(-2):1}"
+file2="${@:(-1):1}"
+args=${@:1:$#-2}
+BASEDIR=$(dirname "$0")
+
+sed -Ef $BASEDIR/prediff.sed < $file1 > $file1.modified
+sed -Ef $BASEDIR/prediff.sed < $file2 > $file2.modified
+
+/usr/bin/diff $args $file1.modified $file2.modified
+exitcode=$?
+
+rm -rf $file1.modified $file2.modified
+
+exit $exitcode
+
```

Comment:
People who want to understand how this works will anyway will try to figure out what the file named `diff` is, so I don't think renaming this to `citus_diff` and having a symlink to it would help much in making it more understandable.

I suggest adding a README file to this directory and having a summary of what this directory is for. What do you think?